Add devMode + clients to init function

barryvdh opened the pull request:
Ask for devMode + ask to add a default client.

barryvdh commented:
Also added the `http://localhost` as default, to make it clear you need to protocol also (for the auth)

kkomelin commented:
I'm not sure if the default *host: http://localhost* will work. As I understand, the **host** requires a hostname and **authHost** needs an url.

barryvdh commented:
I think they both worked, but when authHost is empty, the host option is used, so better to have full url there also

kkomelin commented:
I see. Thanks.

barryvdh commented:
But we could also check both values and parse them to add /remove protocol when needed on initialising the server.

kkomelin commented:
The protocol can be either http or https. It can be difficult to predict what to add when it's absent. I would prefer strict conventions which would be mentioned clearly in the docs and default values.

barryvdh commented:
I removed the host setting, defaulting to `null` (see https://github.com/tlaverdure/laravel-echo-server/issues/111#issuecomment-274032091)
Also removed the questions to verify, but just set the default. And fixed the authPath vs authEndpoint wrong name.

kkomelin commented:
Awesome. Thanks @barryvdh.

kylestev commented on src/echo-server.ts:
```diff
@@ -27,7 +27,7 @@ export class EchoServer {
             }
         },
         devMode: false,
-        host: 'http://localhost',
```
👍 on this line. The host isn't a URI it's a hostname - good catch.

tlaverdure commented on src/cli/cli.ts:
```diff
@@ -62,25 +75,13 @@ export class Cli {
+                message: 'Enter the host of your Laravel authentication server.',
             }, {
-                name: 'authPath',
+                name: 'authEndpoint',
```
@barryvdh Do you think that we should omit this from init? Seems like it will most commonly be set to that path for Laravel devs. Thoughts?

barryvdh replied:
To localhost? Don't think so, I never do that at least.

tlaverdure replied:
ok, I'll merge and update.
